Sources/Yams/Parser.swift

```diff
@@ -125,22 +125,16 @@ public final class Parser {
         yaml_parser_initialize(&parser)
 #if USE_UTF8
         yaml_parser_set_encoding(&parser, YAML_UTF8_ENCODING)
-        utf8CString = string.utf8CString
-        utf8CString.withUnsafeBytes { bytes in
-            let input = bytes.baseAddress?.assumingMemoryBound(to: UInt8.self)
-            yaml_parser_set_input_string(&parser, input, bytes.count - 1)
-        }
+        utf8Slice = string.utf8CString.dropLast()
```

Comment:
Why drop last here but not in UTF-16?

Also the backing for strings changed in Swift 5 to always be UTF-8. Not sure if that matters in this implementation, or should affect our choice of default behavior.

Reply:
Because `utf8CString` has null termination but not in UTF-16. On previous implementation, it passed `bytes.count - 1` to `yaml_parser_set_input_string()`.

> Also the backing for strings changed in Swift 5 to always be UTF-8. Not sure if that matters in this implementation, or should affect our choice of default behavior.

Yup, we'll need to check performance changes about that.